src/webui/www/private/views/preferences.html

```diff
@@ -756,6 +756,10 @@
                 <input type="checkbox" id="csrf_protection_checkbox" />
                 <label for="csrf_protection_checkbox">QBT_TR(Enable Cross-Site Request Forgery (CSRF) protection)QBT_TR[CONTEXT=OptionsDialog]</label>
             </div>
+            <div class="formRow">
+                <input type="checkbox" id="secure_cookie_checkbox" />
```

Comment:
This `id` (not related to public WebAPI) should be in camelcase, we already start using that format for instance: `webUISessionTimeoutInput`.

Reply:
Ok, I was unsure of what style to use, since I saw both. Will change.

Reply:
@Chocobo1 fixed